code/modules/mob/living/carbon/human/death.dm

```diff
@@ -102,21 +104,8 @@
 	set_heartattack(FALSE)
 
 	//Handle species-specific deaths.
-	if(species) species.handle_death(src)
-
-	//Handle brain slugs.
-	var/obj/item/organ/external/head = get_organ("head")
-	var/mob/living/simple_animal/borer/B
-
-	if(istype(head))
-		for(var/I in head.implants)
-			if(istype(I,/mob/living/simple_animal/borer))
-				B = I
-	if(B)
-		if(B.controlling && B.host == src)
-			B.detach()
-
-		verbs -= /mob/living/carbon/proc/release_control

```

Comment:
I believe all the above is no longer needed for borers with the semi-recent refactor.